tests/skipped_tests_gpu.tbl

```diff
@@ -24,6 +24,14 @@ tests/test_linalg.py::test_svd[(2,2)-int32]
 tests/test_linalg.py::test_svd[(2,2)-complex128]
 tests/test_linalg.py::test_svd[(16,16)-complex128]
 //----------------------------------------------------------------------
+// dparray currently do not support given approaches.
```

Comment:
putting comments in this file will brake sorting ability

Reply:
@shssf Thank you for the remark. This file includes another different comments. Let me remove all of them on other refactoring PR.